backend/users/services/contributorsStatsController.js

```diff
@@ -0,0 +1,107 @@
+import { promises as fsPromises } from "fs";
+import path from "path";
+
+// fetch contributors from file
+const fetchContributorsFromFile = async () => {
+  try {
+    const filename = path.join("services", "./../../.all-contributorsrc");
+    const filedata = await fsPromises.readFile(filename, "utf8");
+
+    const data = JSON.parse(filedata);
+    const contributors = data.contributors;
+    return contributors;
+  } catch (error) {
+    console.error("Error reading file or parsing JSON:", error);
+    return null;
+  }
+};
+
+// fetch github info
+const fetchGitHubInfo = async () => {
+  try {
+    const apiUrl = "https://api.github.com/repos/digitomize/digitomize";
```

Comment:
maybe storing the url in an env file would be better instead of hard coding it here